blocked-edges/4.17.16-HostedClusterIsProgressingStuckCondition.yaml

```diff
@@ -0,0 +1,12 @@
+to: 4.17.16
+from: .*
+name: HostedClusterIsProgressingStuckCondition
+url: https://issues.redhat.com/browse/OCPBUGS-51339
```

Comment:
We may now use the nicely written impact statement https://issues.redhat.com/browse/CNTRLPLANE-256 as a reference for more information.

Reply:
👍